core/src/components/datetime/test/prefer-wheel/index.html

```diff
@@ -50,6 +50,14 @@
       locaelInput.onchange = (ev) => {
         datetime.locale = ev.target.value || 'en-US';
       };
+      datetime.isDateEnabled = (dateIsoString) => {
```

Comment:
I'm not sure if this is due to this PR or something else, but I noticed that the disabled dates aren't grayed out: https://user-images.githubusercontent.com/90629384/173372063-340d126a-b344-4e60-8406-b6f6b290cc16.mp4

Reply:
Good catch! looks like I wiped the opacity responsible for this during a merge. I re-added the style and I added a screenshot test for this.